waltz-tools/src/test/java/com/wepay/waltz/tools/storage/StorageCliTest.java

```diff
@@ -183,9 +185,12 @@ public void testListPartition() throws Exception {
     @SuppressWarnings("OptionalGetWithoutIsPresent")
     @Test
     public void testAddAndRemovePartition() throws Exception {
-        int numPartitions = 3;
+        int numPartitions = 5;
         IntegrationTestHelper helper = getIntegrationTestHelper(numPartitions);
-        int partitionId = new Random().nextInt(helper.getNumPartitions());
+        int partitionId1 = new Random().nextInt(helper.getNumPartitions());
+        // ensure partitionId2 is different from partitionId1
+        int partitionId2 = (partitionId1 + 1 + new Random().nextInt(helper.getNumPartitions() - 2)) % helper.getNumPartitions();
```

Comment:
`partitionId2 = (partitionId1 + 1) % helper.getNumPartitions();`
should also be good enough. Just to keep it simple.